lib/events/codes.go

```diff
@@ -326,6 +326,9 @@ const (
 	// generation mismatch code.
 	RenewableCertificateGenerationMismatchCode = "TCB00W"
 
+	// UpgradeWindowStartUpdatedCode is the upgrade windows edit code.
```

Comment:
What about something like "UpgradeWindowStartUpdatedCode is the edit code of UpgradeWindowStartUpdateEvent"?

Reply:
I was following the pattern in the file, but I do think this is way better, thanks!

Reply:
Yeah that's what I try to do as well usually. But we also have a lot of comments in the style of "FooBar is foo bar". Someone pointed out that it might be better to just not write the comment at all if there's nothing more to say than that. ;f So I try to look out for places where we can leave better comments.